src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -56,6 +56,15 @@ public static class MsalError
         /// </summary>
         public const string UserNullError = "user_null";
 
+
+        /// <summary>
+        /// This error code comes back from <see cref="IClientApplicationBase.AcquireTokenSilent(System.Collections.Generic.IEnumerable{string}, IAccount)"/> calls when the 
+        /// <see cref="PublicClientApplication.CurrentBrokerAccount"/> user is passed as the <c>account</c> parameter. Only some brokers (WAM) can login the current user.
+        /// <para>Mitigation</para>
+        /// On Windows, use the broker .WithBroker(true) or use a different account, or otherwise call <see cref="IPublicClientApplication.AcquireTokenInteractive(System.Collections.Generic.IEnumerable{string})"/>
```

Comment:
```suggestion
        /// On Windows, use the broker `.WithBroker(true)` , use a different account, or otherwise call <see cref="IPublicClientApplication.AcquireTokenInteractive(System.Collections.Generic.IEnumerable{string})"/>
```

Reply:
Does ``` work? maybe rather <code>.WithBroker</code> in XML comments?